src/Builder/ListBuilder.php

```diff
@@ -117,45 +134,62 @@ public function fixFieldDescription(AdminInterface $admin, FieldDescriptionInter
                 $fieldDescription->setAssociationMapping($metadata->associationMappings[$fieldDescription->getName()]);
             }
 
-            $fieldDescription->setOption('_sort_order', $fieldDescription->getOption('_sort_order', 'ASC'));
+            $fieldDescription->setOption(
+                '_sort_order',
+                $fieldDescription->getOption('_sort_order', 'ASC')
+            );
         }
 
         if (!$fieldDescription->getType()) {
-            throw new \RuntimeException(sprintf('Please define a type for field `%s` in `%s`', $fieldDescription->getName(), get_class($admin)));
+            throw new \RuntimeException(sprintf(
+                'Please define a type for field `%s` in `%s`',
+                $fieldDescription->getName(),
+                get_class($admin)
+            ));
         }
 
-        $fieldDescription->setOption('code', $fieldDescription->getOption('code', $fieldDescription->getName()));
-        $fieldDescription->setOption('label', $fieldDescription->getOption('label', $fieldDescription->getName()));
+        $fieldDescription->setOption(
+            'code',
+            $fieldDescription->getOption('code', $fieldDescription->getName())
+        );
+        $fieldDescription->setOption(
+            'label',
+            $fieldDescription->getOption('label', $fieldDescription->getName())
+        );
 
         if (!$fieldDescription->getTemplate()) {
             $fieldDescription->setTemplate($this->getTemplate($fieldDescription->getType()));
 
             if ($fieldDescription->getMappingType() == ClassMetadata::MANY_TO_ONE) {
-                $fieldDescription->setTemplate('SonataDoctrinePHPCRAdminBundle:CRUD:list_phpcr_many_to_one.html.twig');
+                $fieldDescription->setTemplate('SonataAdminBundle:CRUD/Association:list_many_to_one.html.twig');
```

Comment:
I copied them all from ORM to us before i went to the step to use them from SonataAdminBundle. So i had exactly the same in here an tested them on our sandbox. Shall i use that branch again on sandbox now?